sklearn/naive_bayes.py

```diff
@@ -700,7 +700,7 @@ class MultinomialNB(_BaseDiscreteNB):
         Class labels known to the classifier
 
     coef_ : ndarray of shape (n_classes, n_features)
-        Mirrors ``feature_log_prob_`` for interpreting MultinomialNB
+        Mirrors ``feature_log_prob_`` for interpreting `MultinomialNB`
         as a linear model.
```

Comment:
Maybe a bit redundant. What about:

```suggestion
        Mirrors ``feature_log_prob_`` for interpreting as a linear model.
```

Reply:
that would seem a bit strange of a sentence to me though

Reply:
Yep, you are right. Something seems missed in the previous sentence.

Another possibility would be:

```suggestion
        Mirrors ``feature_log_prob_`` for interpreting this estimator
        as a linear model.
```

Nevertheless, the original sentence LGTM otherwise.

Reply:
I think it's fine as it is since I did a copy-paste from another class.

Reply:
Then, LGTM!